File: client/src/app/_models/photo.ts
import { Member } from "./member";
import { User } from "./user";

export interface Photo {
    id: number;
    url: string;
    isMain: boolean;
    isCover: boolean;
    title: string;
    appUserId: string;
    appUser: Member;
    created: string;
    publicId: string;
    comments: Comment[];
}

File: API/Entities/Photo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using API.Entities.Comments;

namespace API.Entities
{
    [Table("Photos")]
    public class Photo
    {
        public int Id { get; set; }
        public string PublicId { get; set; }
        public string Url { get; set; }
        public string Title { get; set; }
        public bool IsMain { get; set; }
        public bool IsCover { get; set; }
        public DateTime Created { get; set; } = DateTime.Now;
        public AppUser AppUser { get; set; }
        public int AppUserId { get; set; }
        public List<MainComment> MainComments { get; set; }
    }
}